example/radar-timeline.md

```diff
@@ -0,0 +1,38 @@
+# Radar Timeline
```

Comment:
Since Radar charts can be a little complicated, I would change this example to use a bar chart and title the example "Bar Chart Timeline"